rfcs/text/0012-orchestrator-field-set.md

```diff
@@ -202,23 +202,30 @@ Examples of source data include:
 }
 ```
 
-<!--
-Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
--->
-
 <!--
 Stage 3: Add more real world example source documents so we have at least 2 total, but ideally 3. Format as described in stage 2.
 -->
 
 ## Scope of impact
 
-<!--
-Stage 2: Identifies scope of impact of changes. Are breaking changes required? Should deprecation strategies be adopted? Will significant refactoring be involved? Break the impact down into:
- * Ingestion mechanisms (e.g. beats/logstash)
- * Usage mechanisms (e.g. Kibana applications, detections)
- * ECS project (e.g. docs, tooling)
-The goal here is to research and understand the impact of these changes on users in the community and development teams across Elastic. 2-5 sentences each.
--->
+As this RFC involves the creation of an entirely new fieldset, no breaking
+changes are envisaged. Some existing tooling might need updates to factor in the
+new fieldset's availability, however.
+
+### Ingestion mechanisms
+
+- The Filebeat [Kubernetes processor][5] will need updating, as it currently
+  uses fields that would be out-of-sync with ECS if this is committed.
+- Logstash should see no significant change.
+
+### Usage mechanisms
+
+- Elastic's detection-rules [repo][6] should see no change as there don't appear to
+  be any orchestrator-specific definitions in place.
+
+### ECS project
+
+Documentation updates might be required to reflect the new fieldset.
```

Comment:
@ebeahan I'm not sure if this is actually true or whether all necessary documentation will be auto-generated from the fieldset definitions.

Reply:
@ferozsalam Great question - the additions to the ECS field reference docs will auto-generate from the fieldset definitions once the changes are added.